programmatic-usage/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "example-applications-programmatic-usage",
+  "version": "1.0.0",
+  "description": "Programmatic usage example",
+  "main": "index.js",
+  "scripts": {
+    "test": "node index.js"
```

Comment:
Shouldn't our "test" be invoking Mocha?

Reply:
I thought if somebody was using it programmatically they likely call their script instead of mocha from here. Thoughts?

Reply:
No, it should be:
```json
  "scripts": {
    "test": "mocha"
  }
```
1. the module's only export produces no output, so what's the point?
  * the test script "simple.spec.js" _does_ produce output!
2. it's a _Mocha_ example -- we should _always_ be part of the solution
  * If you're not going to use "simple.spec.js", why include it?

Reply:
In retrospect, you're right. We are invoking Mocha -- just not the normal way.